testing/security/transport_encryption.py

```diff
@@ -6,11 +6,63 @@
 
 
 import sdk_cmd
-
+import sdk_security
+import sdk_utils
 
 log = logging.getLogger(__name__)
 
 
+def setup_service_account(service_name: str,
+                          service_account_secret: str=None) -> dict:
+    """
+    Setup the service account for TLS
+    """
+
+    if sdk_utils.is_open_dcos():
+        log.warn("The setup of a service account requires DC/OS EE")
+        return {}
+
+    name = service_name
+    secret = name if service_account_secret is None else service_account_secret
+    sdk_security.create_service_account(service_account_name=name,
```

Comment:
Does this simply ensure the service account exists? It will already exist in the strict case, yes?

Reply:
The behaviour is to first delete the service account if it exists:
```
log.info('Remove any existing service account and/or secret')
    delete_service_account(service_account_name, service_account_secret)
```
I think this is also related to the fact that in the strict case, we're using a different service account (`'service-acct'`) for the non-TLS tests. Ad discussed with the deletion, I will dig a bit further tomorrow.